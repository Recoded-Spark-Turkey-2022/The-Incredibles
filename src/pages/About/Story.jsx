import React from 'react';
import AboutPHero from '../../assets/pics/aboutpage/aboutp-hero.svg';

function Story() {
  return (
    <section name="story" className="flex flex-col h-screen">
      <div
        name="hero-image"
        className="bg-gradient-to-t from-gray-300 to-transparent max-w-screen h-1/2"
      >
        <img src={AboutPHero} className="w-screen h-full" />
      </div>
      <div name="story" className="max-w-screen h-1/2">
        <h2 className="text-center text-6xl pt-20 pb-10 font-bold text-cyan-700">
          Our Story
        </h2>
        <div className="px-20 mx-20 pb-20">
          <p className="text-center px-20 mx-20 font text-xl max-w-screen">
            It is a long established fact that a reader will be distracted by
            the readable content of a page when looking at its layout. The point
            of using Lorem Ipsum is that it has a more-or-less normal
            distribution of letters, as opposed to using `Content here, content
            here`, making it look like readable English.
          </p>
        </div>
      </div>
    </section>
  );
}

export default Story;
