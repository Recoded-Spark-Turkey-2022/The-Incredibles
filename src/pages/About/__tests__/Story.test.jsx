import React from 'react';
import renderer from 'react-test-renderer';
import Story from '../Story';

describe('Jest Snapshot testing suite', () => {
  it('Matches DOM Snapshot', () => {
    const domTree = renderer.create(<Story />).toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
