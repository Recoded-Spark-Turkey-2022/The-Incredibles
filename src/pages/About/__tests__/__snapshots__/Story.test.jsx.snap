// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<section
  className="flex flex-col h-screen"
  name="story"
>
  <div
    className="bg-gradient-to-t from-gray-300 to-transparent max-w-screen h-1/2"
    name="hero-image"
  >
    <img
      className="w-screen h-full"
      src="aboutp-hero.svg"
    />
  </div>
  <div
    className="max-w-screen h-1/2"
    name="story"
  >
    <h2
      className="text-center text-6xl pt-20 pb-10 font-bold text-cyan-700"
    >
      Our Story
    </h2>
    <div
      className="px-20 mx-20 pb-20"
    >
      <p
        className="text-center px-20 mx-20 font text-xl max-w-screen"
      >
        It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using \`Content here, content here\`, making it look like readable English.
      </p>
    </div>
  </div>
</section>
`;
